Makefile

```diff
@@ -151,25 +151,15 @@ OPERATOR_SETUP = test/operatorsetup
 integ-test: build create-cluster
 
 	echo 'Create CRDs needed by the verrazzano-operator...'
-	kubectl create -f vendor/${VMO_PATH}/${VMO_CRD_PATH}/verrazzano-monitoring-operator-crds.yaml
+	kubectl create -f vendor/${VMO_PATH}/${VMO_CRD_PATH}/verrazzano.io_verrazzanomonitoringinstances_crd.yaml
 
 	echo 'Load docker image for the verrazzano-operator...'
 	kind load docker-image --name ${CLUSTER_NAME} ${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
 
-	echo 'Load docker image needed to install istio CRDs ...'
```

Comment:
these are unrelated to the pusher change correct? were they just not needed all along?

Reply:
Yes, unrelated to the pusher change.  We are currently using istio 1.7.3 instead of istio 1.4.6 and verrazzano-operator does not do anything with istio anymore.